modules/kubecfg/Makefile

```diff
@@ -2,6 +2,10 @@
 
 ## Validate manifests
 kubecfg/validate:
+	@echo "kubecfg/validate"
+	@echo "-------------------------------------------"
 	@@for env in $(shell ls ./rendered/environments) ; do \
-		kubecfg validate ./rendered/environments/$$env/manifests/*.yaml ;\
+		echo "env: $$env" ;\
+		kubecfg validate ./rendered/environments/$$env/manifests/*.yaml || exit 1;\
+		echo "-------------------------------------------" ;\
```

Comment:
if you hit the `exit 1` above you won't get this echo , can you capture the status code of kubecfg , print this echo and then exit ?

Reply:
You mean the `echo -----` ? Does it matter if this is lost? The user will care about the output of `kubecfg`...

Reply:
Then why have it ? :) 

The fact that you had it made me think you cared about the output formatting

Reply:
all the echo statements are there to separate out the environments for which the validation is running for. 

It fails fast, so even if you miss the last `---` it's still clear which environment it was running for.

Reply:
i can do it anyway really, it's easy enough

Reply:
i ended up dropping all the echo stuff. It doesn't add any value.

When there is an error, all the apps (bar pluto) let us know the file/resource which has the problem, which also lets us know the environment.